File: tools/bench/answers/go_haiku/s2/02_vowels.go
package main

import (
	"fmt"
)

func main() {
	s := "the quick brown fox jumps over the lazy dog"
	vowels := "aeiou"
	count := 0
	for _, ch := range s {
		for _, v := range vowels {
			if ch == v {
				count++
				break
			}
		}
	}
	fmt.Println(count)
}

File: tools/bench/answers/rust_haiku/s3/02_vowels.rs
fn main() {
    let s = "the quick brown fox jumps over the lazy dog";
    let vowels = "aeiou";
    let mut count = 0;

    for ch in s.chars() {
        if vowels.contains(ch) {
            count += 1;
        }
    }

    println!("{}", count);
}
